docs/kyma/04-08-update.md

```diff
@@ -36,67 +36,55 @@ In case of dependency conflicts or major changes between components versions, so
 
 - If you update an existing component, make all required changes to the Helm charts of the component located in the [`resources`](https://github.com/kyma-project/kyma/tree/master/resources) directory.
 
-- If you add a new component to your Kyma deployment, add a top-level Helm chart for that component. Additionally, run this command to edit the [Installation custom resource](#custom-resource-installation) and add the new component to the installed components list:
+- If you add a new component to your Kyma deployment, add a top-level Helm chart for that component. Additionally, download the current [Installation custom resource](#custom-resource-installation) on the cluster and add the new component to the components list:
```

Comment:
```suggestion
- If you add a new component to your Kyma deployment, add a top-level Helm chart for that component. Additionally, download the current [Installation custom resource](#custom-resource-installation) to the cluster and add the new component to the components list:
```

Reply:
I would go along the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/download) that says: "the thing the subject is downloaded `to`".

Reply:
ah but then I'd use, download `from` the cluster.